content/app-dev/create-provider/consuming.md

```diff
@@ -36,8 +36,8 @@ We could design this actor to respond to an RPC-style operation called `Checkout
 For the sake of example, let's take a look at what it might look like to respond directly to a checkout operation via actor-to-actor RPC (this is non-compiling psuedocode):
 
 ```rust
-#[no_mangle]
-pub fn wapc_init() {
+#[wasmcloud-actor-core::init]
```

Comment:
We should probably do `actor::init` here because it's easy to alias this.

Reply:
Our previous examples mostly show `actorcore::init`, should we stick with that? Perhaps we should also show the following above the function for clarity:
```
extern crate wasmcloud_actor_core as actorcore;
```

Reply:
Sorry my e-mail didn't link this.. let's do `use wasmcloud_actor_core as actor;` and then `#[actor::init]`

Reply:
Addressed in commit 👍🏻